demo/esri-ds2022/src/components/label/label-content/label-content.tsx

```diff
@@ -48,54 +38,26 @@ export class LabelContent {
   // emit when main panel should be disabled/enabled
   @Event() disableLabelPanel: EventEmitter;
 
-  // todo: split up into multiple states
-  // Need this to rerender on any change since we are making changes to labelclass object, which will not trigger a rerender.
-  @State() reRender = true;
-
-  visibleRangeSlider: HTMLDivElement;
-
   dropdownElement: HTMLCalciteDropdownElement;
 
   dropdownButton: HTMLCalciteButtonElement;
 
   labelStyle: HTMLEsriDs2022LabelContentStyleElement;
 
-  // add widget after render
-  componentDidLoad(): void {
-    this.addScaleRangeSlider();
-  }
+  scaleRangeSlider: __esri.ScaleRangeSlider;
 
-  // Called after every re-render. Not called on initial draw.
+  scaleRangeSliderWatch: __esri.WatchHandle;
 
-  componentDidUpdate(): void {
-    this.internalLabelUpdated.emit();
-  }
+  mapViewScaleWatch: __esri.WatchHandle;
 
-  @Listen("closeLabelPopovers", { target: "window" })
-  closeLabelPopoversHandler(): void {
-    if (this.labelStyle) {
-      document.body.removeChild(this.labelStyle);
-      this.labelStyle = null;
-    }
-    this.disableLabelPanel.emit(false);
-  }
-
-  getDisplayFieldName(): string {
-    return (this.labelClass as any).getLabelExpressionSingleField();
-  }
-
-  addScaleRangeSlider(): void {
-    // todo: create this on connectedCallback and destroy on disconnectedCallback.
-    // todo: Watch props and set them to the slider instead of on componentDidLoad
-    const scaleRangeSlider = new ScaleRangeSlider({
-      container: this.visibleRangeSlider,
+  componentWillLoad() {
```

Comment:
Can we create the slider on `connectedCallback` instead of `componentWillLoad`?

Reply:
For some reason using connectedCallback leads to this
![Screen Shot 2022-02-24 at 10 15 44 PM](https://user-images.githubusercontent.com/14285436/155664602-8fb117fb-0baf-49b8-a26d-ece15d5856df.png)